test/index.js

```diff
@@ -70,6 +70,58 @@ test('when short option withValue used without value then stored as flag', funct
   t.end();
 });
 
+test('short option group behaves like multiple short options', function(t) {
+  const passedArgs = ['-rf'];
+  const passedOptions = { };
+  const expected = { flags: { r: true, f: true }, values: { r: undefined, f: undefined }, positionals: [] };
+  const args = parseArgs(passedArgs, passedOptions);
+
+  t.deepEqual(args, expected);
+
+  t.end();
+});
+
+test('short option group does not consume subsequent positional', function(t) {
+  const passedArgs = ['-rf', 'foo'];
+  const passedOptions = { };
+  const expected = { flags: { r: true, f: true }, values: { r: undefined, f: undefined }, positionals: ['foo'] };
+  const args = parseArgs(passedArgs, passedOptions);
+  t.deepEqual(args, expected);
+
+  t.end();
+});
+
+// Refs: https://github.com/yargs/yargs-parser/blob/ae11f496a8318ea8885aa25015d429b33713c314/test/yargs-parser.cjs#L30
```

Comment:
I don't think the Yargs reference is canonical enough to include in code. A better canonical reference is:

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html

> Guideline 5:
> One or more options without option-arguments, followed by at most one option that takes an option-argument, should be accepted when grouped behind one '-' delimiter.